yml/dp02_dc2.yaml

```diff
@@ -7090,692 +7090,692 @@ tables:
   - name: iPSFluxMin
     "@id": "#DiaObject.iPSFluxMin"
     datatype: double
     mysql:datatype: DOUBLE
-    description:
+    description: Minimum diaSource PSF flux
   - name: iPSFluxNdata
     "@id": "#DiaObject.iPSFluxNdata"
     datatype: double
     mysql:datatype: DOUBLE
-    description:
+    description: The number of data points used to compute iPSFluxChi2
   - name: iPSFluxPercentile05
     "@id": "#DiaObject.iPSFluxPercentile05"
     datatype: double
     mysql:datatype: DOUBLE
-    description:
+    description: 5th percentile diaSource PSF flux
   - name: iPSFluxPercentile25
     "@id": "#DiaObject.iPSFluxPercentile25"
     datatype: double
     mysql:datatype: DOUBLE
-    description:
+    description: 10th percentile diaSource PSF flux
   - name: iPSFluxPercentile50
     "@id": "#DiaObject.iPSFluxPercentile50"
     datatype: double
     mysql:datatype: DOUBLE
-    description:
+    description: Median diaSource PSF flux
   - name: iPSFluxPercentile75
     "@id": "#DiaObject.iPSFluxPercentile75"
     datatype: double
     mysql:datatype: DOUBLE
-    description:
+    description: 75th percentile diaSource PSF flux
   - name: iPSFluxPercentile95
     "@id": "#DiaObject.iPSFluxPercentile95"
     datatype: double
     mysql:datatype: DOUBLE
-    description:
+    description: 95th percentile diaSource PSF flux
   - name: iPSFluxSigma
     "@id": "#DiaObject.iPSFluxSigma"
     datatype: double
     mysql:datatype: DOUBLE
-    description:
+    description: Standard deviation of the distribution of iPSFlux
   - name: iPSFluxSkew
     "@id": "#DiaObject.iPSFluxSkew"
     datatype: double
     mysql:datatype: DOUBLE
-    description:
+    description: Skew of diaSource PSF flux
   - name: iPSFluxStetsonJ
     "@id": "#DiaObject.iPSFluxStetsonJ"
     datatype: double
     mysql:datatype: DOUBLE
-    description:
+    description: StetsonJ statistic of diaSource PSF flux
   - name: iTOTFluxMean
     "@id": "#DiaObject.iTOTFluxMean"
     datatype: double
     mysql:datatype: DOUBLE
-    description:
+    description: Weighted mean of the PSF flux forced photometered at the diaSource position on the calibrated image
   - name: iTOTFluxMeanErr
     "@id": "#DiaObject.iTOTFluxMeanErr"
     datatype: double
     mysql:datatype: DOUBLE
-    description:
+    description: Standard error on iTOTFluxMean
   - name: iTOTFluxSigma
     "@id": "#DiaObject.iTOTFluxSigma"
     datatype: double
     mysql:datatype: DOUBLE
-    description:
+    description: Standard deviation of the PSF flux forced photometered at the diaSource position on the calibrated image
   - name: nDiaSources
     "@id": "#DiaObject.nDiaSources"
     datatype: long
     mysql:datatype: BIGINT
-    description:
+    description: Number of diaSources associated with this diaObject
   - name: pixelId
     "@id": "#DiaObject.pixelId"
     datatype: double
     mysql:datatype: DOUBLE
-    description:
+    description: HtmIndex20 of ra, decl coordinate
```

Comment:
of ra,dec coordinate

Reply:
Even if I'm referring to the columns in this table named `ra, decl`

Reply:
Yeah, I'm torn on this one: Maybe leave it decl, but make sure we note to fix the description in future files?